### PR TITLE
Ad Duration values: round to the nearest integer to compensate for du…

### PR DESCRIPTION
**Pull request by SInCE**
Example: Ad duration is 30.04 seconds, 30 seconds to be expected to be shown in the UI.

Action taken: round ad duration to the closest integer value.

### CODE REVIEW

**Comment by hanneslinder** [comment]
Change looks good, but can you please add a `changelog` as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```
Comment (hanneslinder) [inline CHANGELOG.md:7]:
```suggestion
## [develop]
```

We usually use `develop` for unreleased changes :smiley:

File: src/ts/stringutils.ts [hunk 2]
```diff
@@ -106,12 +106,12 @@ export namespace StringUtils {
...
+          time = Math.round(player.getDuration() - player.getCurrentTime());
         }
       } else if (formatString.indexOf('playedTime') > -1) {
-        time = player.getCurrentTime();
+        time = Math.round(player.getCurrentTime());
       } else if (formatString.indexOf('adDuration') > -1) {
-        time = player.getDuration();
+        time = Math.round(player.getDuration());
       }
       return formatNumber(time, formatString);
```
Comment (ravloony) [inline src/ts/stringutils.ts:116]: Nit: You could do a single `Math.round` call here instead and save a few bytes.
```suggestion
      return formatNumber(Math.round(time), formatString);
```

**Review by ravloony — CHANGES_REQUESTED** [review]
Just a nit, otherwise looks fine.

File: src/ts/stringutils.ts (hunk 2 above)
Comment (SInCE) [reply src/ts/stringutils.ts]: 👍

**Review by hanneslinder — APPROVED** [review]
LGTM :+1:

**Review by Cagince — APPROVED** [review]
LGTM as well

**Review by ravloony — APPROVED** [review]
LGTM :+1:

**Pull request merged**